hadoop-cloud-storage-project/hadoop-huaweicloud/pom.xml

```diff
@@ -173,8 +173,8 @@
     </dependency>
     <dependency>
       <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
-      <version>1.7.4</version>
+      <artifactId>powermock-api-mockito2</artifactId>
```

Comment:
move the declaration of this jar into the hadoop-project pom with this module importing it, same for the one below. There's an existing powermock.version they can use once all projects are in sync